pipeline/pipeline.go

```diff
@@ -0,0 +1,33 @@
+package pipeline
+
+import (
+	"os"
+	"os/exec"
+
+	"github.com/m-lab/clinic2019/csvParser"
+
+	"github.com/m-lab/clinic2019/incident_viewer_demo/incident"
+)
+
+type IncidentProducer interface {
+	FindIncidents()
+}
+
+type AnalyzerIncidents struct{}
+
+func (ai *AnalyzerIncidents) findIncidents() {
+	// I think this will generate the CSV without needing a cmd.Run()
+	exec.Command("bash", "-c", "go run github.com/m-lab/signal-searcher  | sort -nk1 > incidents.csv").Output()
+}
+
+func runPipeline() []incident.DefaultIncident {
```

Comment:
where is this function returning an array of DefaultIncidents

Reply:
This is outdated